test/node/decimal128_tests.js

```diff
@@ -808,29 +807,31 @@ exports['fromString exponent normalization'] = function(test) {
   );
   test.deepEqual(bytes, result.bytes);
 
+  // this should throw error according to spec.
```

Comment:
and it's covered by the corpus tests?

Reply:
yep, one of the cases that should throw a parse error here: https://github.com/mongodb/js-bson/pull/218/files?diff=unified#diff-50fc67aa335868ab2ed1e27d8492655bR168